jest.unit.config.js

```diff
@@ -18,5 +18,5 @@ module.exports = {
     testEnvironment: 'jsdom',
     testMatch: ['**/__tests__/*-test.tsx'],
     testEnvironmentOptions: {url: 'http://test.tuenti.com'},
-    setupFilesAfterEnv: [require.resolve('./setup-test-env.tsx'), '@testing-library/jest-dom/extend-expect'],
+    setupFilesAfterEnv: ['<rootDir>/setup-test-env.tsx'],
```

Comment:
the `setup-test-env` now imports the jest-dom module
https://github.com/testing-library/jest-dom?tab=readme-ov-file#usage